tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/dashboards/Instructor_Dashboard.yaml

```diff
@@ -546,20 +546,8 @@ position:
     children: []
     id: MARKDOWN-904VlyNSAd
     meta:
-      code: '### Course Enrollment
-
-
-
-        These charts can help you understand how many students are enrolled in your
-        course by showing how enrollments have changed over time and which enrollment
-        modes students are choosing. More details about each chart can be found by
-        clicking the "3 dot" menu at the top right of the chart and selecting "Show
-        chart description".
-
-
-        You can filter these results by choosing selecting various options from the
-        filters on the left.'
-      height: 20
+      code: <div>Filter these results by selecting various options from the filters panel.</div>
+      height: 7
```

Comment:
FYI wrapping this text in a `<div>` makes it take up less height than leaving it unwrapped, because markdown automatically wraps it in a `<p>` otherwise.

Reply:
Also controversial: I removed "on the left" from this text because if we're using Superset with RTL languages, the panel will likely be on the right. But feel free to shoot this down :)